Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2557,13 +2557,16 @@ def _http_request(self, method, url_suffix, full_url=None, headers=None,
                 if not self._is_status_code_valid(res, ok_codes):
                     err_msg = 'Error in API call [{}] - {}' \
                         .format(res.status_code, res.reason)
+                    if 'text/plain' in res.headers.get('Content-type'):
```

Comment:
i think we can just append `res.text` in the `except` clause, regardless of the content type
what do you say?

Reply:
I think not, because if the content type is `json` it will not be a readable string it will be a `json.dumps` string.
And the header should be a good indication of whether it's a text format or a json format string.

Reply:
yea but it was to be a valid json, then it wouldnt fail when trying to deserialize it..
im ok with it not being readable, but its better than the nothing we return now